lib/internal/errors.js

```diff
@@ -1182,6 +1182,21 @@ E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s', TypeError);
 E('ERR_UNKNOWN_FILE_EXTENSION', 'Unknown file extension: %s', TypeError);
 E('ERR_UNKNOWN_MODULE_FORMAT', 'Unknown module format: %s', RangeError);
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s', TypeError);
+E('ESM_CJS_LOAD_ERROR', (error) => {
+
+  if(error instanceof SyntaxError && (/^import/mi).test(error.stack)){
+    const sp =  error.stack.split('\n');
+    const filepath = sp[0];
+    const line = `${sp[1]}\n${sp[2]}`
+    return `${filepath}
+${line}
+Warning: Attempted to load commonjs module but found static import statement
+Further information can be found at https://nodejs.org/api/esm.html
+SyntaxError: ${error.message}`
```

Comment:
the `error.stack` in this error contains the string `SyntaxError: ... ` again .. I don't know why..

Reply:
The text here is the error "message", where the text provided when logged to the console is a combination of `e.message` and `e.stack`.

Rather than constructing a new error as you are doing here, I would suggest moving all this code into `loader.js` as I've suggested below.